go/vt/sqlparser/ast.go

```diff
@@ -1364,7 +1364,15 @@ type ColumnType struct {
 
 // ColumnTypeOptions are generic field options for a column type
 type ColumnTypeOptions struct {
+	/* We need both Null and NotNull to distinguish 3 cases -
```

Comment:
I think we should us a `*bool` instead of two bools.

Reply:
Look at `Select.cache` for another example of the same logic

Reply:
done